automated/linux/ota-update/download-update.sh

```diff
@@ -152,14 +152,11 @@ if [ "${TYPE}" = "uboot" ]; then
 fi
 
 # wait for 'install-post' signal
-SIGNAL=$(</var/sota/ota.signal)
-while [ ! "${SIGNAL}" = "install-post" ]
+while ! grep "install-post" /var/sota.ota.signal
```

Comment:
should probably be '/var/sota/ota.signal'

Reply:
Yes, good catch. Fixed.